### PR TITLE
Update or create project

### PR DESCRIPTION
**Pull request by philippfranke**

### CODE REVIEW

File: tests/files/letschat_variable.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    "services": [
+        {
+            "id": "frontend",
+            "apps": [
+                {
+                    "id": "nide",
+                    "domain": {
+                        "uri": "letschat.sloppy.zone"
+                    },
+                    "mem": $memory,
```
Comment (jzaefferer) [inline tests/files/letschat_variable.json]: Shouldn't these be in double quotes as well?

File: tests/files/letschat_variable.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+                }
+            ]
+        },
+        {
+            "id": "backend",
+            "apps": [
+                {
+                    "id": "mongodb",
+                    "mem": 512,
+                    "image": "mikemichel/lets-chat",
```
Comment (jzaefferer) [inline tests/files/letschat_variable.json]: Doesn't really matter: wrong image

File: src/command/change.go [hunk 3]
```diff
@@ -16,14 +16,18 @@ import (
...
+		api.ProjectsCreater
+	}
+	Apps api.AppsUpdater
 }
 
 // Help should return long-form help text.
 func (c *ChangeCommand) Help() string {
 	helpText := `
-Usage: sloppy change [OPTIONS] (PROJECT/SERVICE/APP | PROJECT FILENAME)
+Usage: sloppy change [OPTIONS] (PROJECT/SERVICE/APP | FILENAME)
```
Comment (jzaefferer) [inline src/command/change.go]: Can we make sure the old syntax (sloppy change PROJECT FILENAME) still works after this change? Or at least throw an explicit error about the change? If the latter, we should still do a major version bump.

Comment (philippfranke) [inline src/command/change.go]: Good catch.

File: tests/files/letschat_variable.json (hunk 1 above)
Comment (philippfranke) [inline tests/files/letschat_variable.json]: `$memory` is an integer so it doesn't need to be in double quotes.

**Comment by jzaefferer** [comment]
Looks good

**Pull request merged**